packages/shared/styles/components/molecules/SfCallToAction.scss

```diff
@@ -2,64 +2,84 @@
 
 .sf-call-to-action {
   display: flex;
+  box-sizing: border-box;
   flex-direction: var(--call-to-action-flex-direction, column);
-  align-items: center;
-  align-content: center;
-  justify-content: var(--call-to-action-justify-content);
-  padding: var(
-    --call-to-action-padding,
-    var(--spacer-big) var(--spacer-medium)
-  );
-  background-color: var(--call-to-action-background, var(--c-black));
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: cover;
-  color: var(--call-to-action-color, var(--c-on-dark));
+  align-items: var(--call-to-action-align-items);
+  justify-content: var(--call-to-action-justify-content, left);
+  min-height: var(--call-to-action-min-height, 16.75rem);
+  padding: var(--call-to-action-padding, var(--spacer-base));
+  background-position: var(--call-to-action-background-position, center);
```

Comment:
I think you can make a shortcut out of all background properties: https://developer.mozilla.org/pl/docs/Web/CSS/background :)

Reply:
I can do it, but I prepared also the issue for creating mixin for this. https://github.com/DivanteLtd/storefront-ui/issues/865 
Should I change that or prepare mixin? What do you think @aniamusial @pspaczek ?